Doc/c-api/init.rst

```diff
@@ -919,8 +919,12 @@ from a C thread is::
 Note that the ``PyGILState_*`` functions assume there is only one global
 interpreter (created automatically by :c:func:`Py_Initialize`).  Python
 supports the creation of additional interpreters (using
-:c:func:`Py_NewInterpreter`), but mixing multiple interpreters and the
-``PyGILState_*`` API is unsupported.
+:c:func:`Py_NewInterpreter`), but switching between interpreters via the
+``PyGILState_*`` API is unsupported.  Similiarly, after creating a
+sub-interpreter via :c:func:`!PyGILState_Ensure`, interacting with
+the GIL of other interpreters (including, but not limited to trying
```

Comment:
```suggestion
the GIL of other interpreters (including, but not limited to, trying
```

Minor nitpick. I am not an English native, but I should have put a comma after "to" (sorry for my previous suggestion missing it). Alternatively, I think it is also possible to have no comma at all (i.e. neither after "including" nor after "to").

Reply:
I like commas when writing, they help keep "flow." I'm not too familiar with how the docs are supposed to be written, though -- maybe it's better to avoid them.